CHANGELOG.md

```diff
@@ -23,7 +23,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- `AccountManager::get_participation_event_ids` method;
```

Comment:
This was still added, to the account, so it would be nice to keep mentioning it.